lib/rubocop/cop/correctors/empty_line_corrector.rb

```diff
@@ -2,24 +2,24 @@
 
 module RuboCop
   module Cop
-    # This class does empty line auto-correction
-    class EmptyLineCorrector
-      class << self
```

Comment:
I think `class << self` is an acceptable case.

Reply:
But what about if that contains only class method definitions or constants?

Reply:
Hmm, In any case, I’m wondering what the merit is to change class to module.

Reply:
For me, module adds to explicitness that this is more a class of utility methods and therefore it does not make sense to create instances of this class. 
Maybe an author of this section of styleguide had more reasons?

Reply:
Yeah, I think it's agree if it's designed as a module :-) However modifying an existing implementation designed as a class can have unintended potential breaking changes. So, especially when it's published API. I’m worried that this enforced style seems too strong to be applied by default.

Reply:
Yes, I marked it as unsafe and briefly mentioned in the description of the cop why.
So are you suggesting to make it disabled? If yes, then should it be also disabled for `rubocop`s own codebase?

Reply:
> So are you suggesting to make it disabled? If yes, then should it be also disabled for rubocops own codebase?

Yes. I'm worried that style preferences will affect an existing design. I'm not sure that it will be preferred to be applied in existing development by default. And it seems to be no reason to enable it on RuboCop’s codebase only.
OTOH, it may be possible to create some condition for it to be enabled by default in the future. However, it seems that the current behavior is a stronger warning than the explanation in the style guide.